JitsiConference.ts

```diff
@@ -175,7 +175,7 @@ export interface IStopSessionOptions {
     sendSessionTerminate?: boolean;
 }
 
-const logger = getLogger('JitsiConference');
+const logger = getLogger('Core:JitsiConference');
```

Comment:
No strong opinion, question: what capitalization do we like here? lowercase except for acronyms like RTC? Actually all lower sounds ok? Minus the last part, there we use the full component name.